static/styles/darkMode.css

```diff
@@ -1,4 +1,4 @@
-/* darkmode.css */
+@import url('https://fonts.googleapis.com/css?family=Poppins:400,700,900');
```

Comment:
this will not work in an closed network you should provide the nesscary files instead of fetching them online...

Reply:
i need to do it locally ??